components/kyma-environment-broker/internal/broker/client.go

```diff
@@ -106,53 +106,53 @@ func (c *Client) Deprovision(instance internal.Instance) (string, error) {
 }
 
 // SendExpirationRequest requests Runtime suspension due to expiration
-func (c *Client) SendExpirationRequest(instance internal.Instance) (string, error) {
+func (c *Client) SendExpirationRequest(instance internal.Instance) (suspensionUnderWay bool, err error) {
 	request, err := preparePatchRequest(instance, c.brokerConfig.URL)
 	if err != nil {
-		return "", err
+		return false, err
 	}
 
 	resp, err := c.httpClient.Do(request)
 	if err != nil {
-		return "", errors.Wrapf(err, "while executing request URL: %s", request.URL)
+		return false, errors.Wrapf(err, "while executing request URL: %s for instanceID: %s", request.URL, instance.InstanceID)
 	}
 	defer c.warnOnError(resp.Body.Close)
 
-	return processResponse(resp.StatusCode, resp)
+	return processResponse(instance.InstanceID, resp.StatusCode, resp)
 }
 
-func processResponse(statusCode int, resp *http.Response) (string, error) {
+func processResponse(instanceID string, statusCode int, resp *http.Response) (suspensionUnderWay bool, err error) {
 	switch statusCode {
 	case http.StatusAccepted, http.StatusOK:
 		{
-			log.Infof("Request accepted with status: %+v", statusCode)
+			log.Infof("Request for instanceID: %s accepted with status: %+v", instanceID, statusCode)
 			operation, err := decodeOperation(resp)
 			if err != nil {
-				return "", err
+				return false, err
 			}
-			log.Infof("operation: %+v", operation)
-			return operation, nil
+			log.Infof("For instanceID: %s we received operation: %s", instanceID, operation)
+			return true, nil
 		}
 	case http.StatusUnprocessableEntity:
 		{
-			log.Warnf("Entity unprocessable - request rejected with status: %+v", statusCode)
+			log.Warnf("For instanceID: %s we received entity unprocessable - status: %+v", instanceID, statusCode)
 			description, errorString, err := decodeErrorResponse(resp)
 			if err != nil {
-				return "", err
+				return false, errors.Wrapf(err, "for instanceID: %s", instanceID)
 			}
-			log.Warnf("error: %+v description: %+v", errorString, description)
-			return "", nil
+			log.Warnf("error: %+v description: %+v instanceID: %s", errorString, description, instanceID)
+			return false, nil
 		}
 	default:
 		{
 			if statusCode >= 200 && statusCode <= 299 {
```

Comment:
200 and 204 will be handled in the first `case`. We expect 200, 204 and 422 (second `case` and any other code is `unexpected`.  In case we change set of expected status code we would need to adjust not only case guard but also the if condition.